sci-mathematics/coq/coq-8.9.1.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+inherit eutils multilib
+
+MY_PV=${PV/_p/pl}
+MY_P=${PN}-${MY_PV}
+
+DESCRIPTION="Proof assistant written in O'Caml"
+HOMEPAGE="http://coq.inria.fr/"
+SRC_URI="https://github.com/coq/coq/archive/V${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="gtk debug +ocamlopt doc"
+
+RDEPEND="
+	>=dev-lang/ocaml-3.11.2:=[ocamlopt?]
+	>=dev-ml/camlp5-6.02.3:=[ocamlopt?]
+	gtk? ( >=dev-ml/lablgtk-2.10.1:=[sourceview,ocamlopt?] )"
+DEPEND="${RDEPEND}
+	dev-ml/findlib
+	doc? (
+		media-libs/netpbm[png,zlib]
```

Comment:
Could you sort this doc ( ) list while at it, dev* before media-libs and virtual.